frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -40,6 +41,10 @@ common_settings.screen = {
         require("ui/elements/refresh_menu_table"),
     },
 }
+if Device:isAndroid() then
+    table.insert(common_settings.screen.sub_item_table, require("ui/elements/screen_fullscreen_menu_table"))
```

Comment:
@chrox I didn't know. Thanks.

So once those PR's are merged I building android on Ubuntu 16.04 should work right?
I think I'll hold on until then